mobile/cmp/dimensionChooser/DimensionChooser.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {HoistComponent, elemFactory} from '@xh/hoist/core';
+import PT from 'prop-types';
+
+import {div} from '@xh/hoist/cmp/layout';
+import {button} from '@xh/hoist/mobile/cmp/button';
+import {dialog} from '@xh/hoist/mobile/cmp/dialog';
+import {Icon} from '@xh/hoist/icon';
+import {select} from '@xh/hoist/mobile/cmp/form';
+
+import {size, isEmpty} from 'lodash';
+
+import './DimensionChooser.scss';
+import {DimensionChooserModel} from '@xh/hoist/desktop/cmp/dimensionchooser';
```

Comment:
This would have to be moved to 'all'.  We'll never want the mobile package importing from the desktop package.

interesting pattern here, where we are able to share a model cross-platform, but not the view.  I think we have seen this elsewhere, and can expect to see it again.  It ends up being a bit strange for the application, which needs to import the two different bits of the component from different locations.  I wonder if a re-export of the model in the mobile  is in order.  

Alternatively, we could look to make the Component cross-platform.  How different are they?

Reply:
Yes, I think we're partly just lucky that the desktop patterns fit neatly into the mobile dialog api. The main render methods are nearly identical. However I think there are enough context-specific helper methods/components that making it cross-platform would be confusing.